### DISCUSSION TITLE
Angular MFE example loading a remote in a lib child route

### DISCUSSION THREAD

**Discussion by gkamperis** [nrwl/nx#10599] [General]
Hello everyone!

I have been looking all around for MFE examples and every single one shows the obvious host/remote app integration in the main top level angular outlet. 

I have used Nx 14 and a  shell + remote app configs and it seems to work ok.

Within a monorepo it is super common to have lazy loaded libs with routes defined at lib level and with outlets defined in the llib's components.

How would I show a component defined in a remote inside a lazy loaded lib that defines child routes and in a router-outlet of the lib's nested child component? 

Thanks in advance for any suggestions.

**Comment by kenatwex** [comment]
Im seeking this very real-world best-case best-practice de-coupled way myself

https://github.com/nrwl/nx/issues/11954

seems this mechanism has been overlooked

hoping for workaround asap
